# Clade I Mpox virus genomic diversity in the Democratic Republic of the Congo, 2018 - 2024: Predominance of Zoonotic Transmission #


## ABOUT ##

**429** newly generated **MPXV Clade-I** genomes from **DRC** from **2018 to 2024**.


## STATUS ##

We are pleased to let you know that our manuscript is now accepted in principle at **Cell**.  


## DATA ##

**NCBI**: [NCBI](https://www.ncbi.nlm.nih.gov/genbank/)  
**Embargo:** The data will be publicly available on NCBI after [2025-03-12] or upon publication of the associated article.  

**Git**: [https://forge.ird.fr/transvihmi/nfernandez/Clade-I_Mpox-virus_genomic-diversity_DRC](https://forge.ird.fr/transvihmi/nfernandez/Clade-I_Mpox-virus_genomic-diversity_DRC)  

## LICENSE ##

**Intellectual property** belongs to INRB and all involved partners.

The **data** and **associated files** are made available under a Creative Commons Attribution-NoDerivatives 4.0 International license  
**[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/)**

### >>> Key Points ###

- **Attribution**: You must give appropriate credit, provide a link to the license, and indicate if changes were made.
- **NonCommercial**: You may not use the material for commercial purposes.
- **NoDerivatives**: If you remix, transform, or build upon the material, you may not distribute the modified material.  
                     The work can only be shared in its original form, without any alterations.

_For more details, see the full license [here](https://creativecommons.org/licenses/by/4.0)._


### >>> Preprint ###

The copyright holder for this preprint is the author/funder, who has granted **medRxiv** a license to display the preprint in perpetuity. It is made available under a **[CC-BY 4.0](https://creativecommons.org/licenses/by/4.0)** international license.  


### >>> Data ###

**IMPORTANT:** Data are released **only for public health usage**!  
**Any use** in commercial, academic publications, or other purposes **requires prior authorization from the authors**!



## CITATION ##

### Cell ###
**Clade I Monkeypox virus genomic diversity in the Democratic Republic of the Congo, 2018 - 2024: Predominance of Zoonotic Transmission**  
**DOI**:[]()  
**Print**:[]()  

### BioRxiv ###
**Clade I Mpox virus genomic diversity in the Democratic Republic of the Congo, 2018 - 2024: Predominance of Zoonotic Transmission**  
**DOI**: [https://doi.org/10.1101/2024.08.13.24311951](https://doi.org/10.1101/2024.08.13.24311951)  
**Preprint**: [https://www.medrxiv.org/content/10.1101/2024.08.13.24311951v1.full.pdf](https://www.medrxiv.org/content/10.1101/2024.08.13.24311951v1.full.pdf)  


## CORRESPONDING AUTHORS ##

- Eddy Kinganda-Lusamaki [eddylusamaki@gmail.com](eddylusamaki@gmail.com)
- Martine Peeters [martine.peeters@ird.fr](martine.peeters@ird.fr)


## AUTHORS ##

**Eddy Kingada Lusamaki** ; Adrienne Amuri Aziza ; Gradi Luakanda Ndelemo ; Emmanuel Hasivirwe Vakaniaki ; Nicolas Fernandez Nunez ; Catherine B. Pratt ;
Prince Akil Bandali ; Jean-Claude Makangara Cigolo ; Nicole A. Hoff ; Sabin Sabiti Nundu ; Noella Mulopo Mukanya ; Michel Ngimba ; Brigitte Modrada Madakpa ;
Ruth Diavita ; Princesse Paku Tshambu ; Elisabeth Pukuta Simbu ; Sydney Merritt ; Aine O'Toole ; Nicola Low ; Antoine Nkuba Ndaye ; Hugo Kavunga Membo ;
Robert Shongo ; Emile Malembi ; Thierry Kalonji ; Cris Kacita ; Emmanuel Lokilo Lofiko ; Sifa Kavira Bapitani ; Ola Rilia ; Pauline-Chloe Muswamba ;
Hugo Kavunga ; Membo Francisca ; Muyembe Mawete ; Raphael Lumembe ; Gabriel Kabamba ; Michel Kenye ; Andre Citenga ; Fiston Cikaya ; Zadok Yungwe ;
Meris Matondo ; Francois Kasongo ; Paul Tshiminyi ; Junior Bulabula ; Frida Nkawa ; Emile Muhindo Milonde ; Adele Kavira Kamaliro ; Zephanie Paluku Kalimuli ;
Jeriel Mubalaa Mufungizi ; Tavia Bodisa Matamu ; Mike R. Wiley ; Elise Devos ; Eugene Bangwen ; Isabel Brosius ; Edyth Parker ; Genay Pilarowski ;
Laurens Liesenborghs ; Tony Wawina Bokalanga ; Koen Vercauteren ; Daniel Mukadi Bamuleka ; Lorenzo Subissi ; Jean-Jacques Muyembe ; Jason Kindrachuk ;
Emilande Guichet ; Ahidjo Ayouba ; Andrew Rambaut ; Eric Delaporte ; Sofonias K. Tessema ; Eric D'Ortenzio ; Anne W. Rimoin ; Lisa E. Hensley ;
Placide Mbala Kingebeni ; Martine Peeters ; Steve Ahuka Mundeke

## Rulegraph ##

<img src="./scripts/auspice-augur_snakemake-workflow/resources/visuals/augur_rulegraph.png" width="100" height="300">  

<img src="./scripts/auspice-augur_snakemake-workflow/resources/visuals/augur_filegraph.png" width="200" height="600">  

## Auspice ##

**Drag & Drop** [auspice.json](./results/auspice-augur_snakemake-workflow/auspice/auspice_Clade-I_Mpox-virus_genomic-diversity_DRC.json) here: [asupice.us](https://auspice.us)


## Git map ##

```shell
 ğŸ§© GIT/
 â”œâ”€â”€ ğŸ“š README.md [READ FIRST]
 â”œâ”€â”€ ğŸªª LICENSE
 â”œâ”€â”€ ğŸš« .gitignore
 â”œâ”€â”€ ğŸ“‚ .git/
 â”œâ”€â”€ ğŸ“‚ data/
 â”‚   â”œâ”€â”€ ğŸ“Š  sequences_metadata.tsv [COMING SOON]
 â”‚   â”œâ”€â”€ ğŸ“‚ sequences_fasta_pre-release/ [429 newly sequenced]
 â”‚   â”‚   â””â”€â”€ ğŸ§¬ {SAMPLE}.fasta
 â”‚   â””â”€â”€ ğŸ“‚ auspice-augur_snakemake-workflow/ [fasta and metadata for auspice figure]
 â”‚       â”œâ”€â”€ ğŸ“Š  metadata_Clade-I_Mpox-virus_genomic-diversity_DRC.tsv
 â”‚       â””â”€â”€ ğŸ§¬ sequences_Clade-I_Mpox-virus_genomic-diversity_DRC.fasta
 â”œâ”€â”€ ğŸ“‚ publications/
 â”‚   â””â”€â”€ ğŸ“ Pre-print\_bioRxiv\_Clade-I\_Mpox-virus\_genomic-diversity\_DRC\_2018-2024\_Predominance-Zoonotic-Transmission.pdf
 â”œâ”€â”€ ğŸ“‚ results/
 â”‚   â”œâ”€â”€ ğŸ“‚ apobec3_analysis/
 â”‚   â”‚   â”œâ”€â”€ ğŸ§¬ 530mpox429_98_3.fasta
 â”‚   â”‚   â”œâ”€â”€ ğŸŒ³ 530mpox429_98_3.tree
 â”‚   â”‚   â”œâ”€â”€ ğŸ§¬ 530mpox429_98_3.aln.fasta
 â”‚   â”‚   â”œâ”€â”€ ğŸŒ³ 530mpox429_98_3.aln.fasta.iqtree
 â”‚   â”‚   â”œâ”€â”€ ğŸŒ³ 530mpox429_98_3.aln.fasta.treefile
 â”‚   â”‚   â”œâ”€â”€ ğŸ“Š 530mpox429_98_3.tree.amino_acid.reconstruction.csv
 â”‚   â”‚   â”œâ”€â”€ ğŸ“Š 530mpox429_98_3.tree.branch_snps.reconstruction.csv
 â”‚   â”‚   â”œâ”€â”€ ğŸ“Š 530mpox429_98_3.tree.branch_snps.reconstruction.csv
 â”‚   â”‚   â”œâ”€â”€ ğŸ“ 530mpox429_98_3.aln.fasta.log
 â”‚   â”‚   â””â”€â”€ ğŸŒ€ 530mpox429_98_3.aln.fasta.{EXT}
 â”‚   â”œâ”€â”€ ğŸ“‚ ML_tree/
 â”‚   â”‚   â”œâ”€â”€ ğŸ§¬ 489mpoxsup90_348_98_43.aln.fasta
 â”‚   â”‚   â”œâ”€â”€ ğŸŒ³ 489mpoxsup90_348_98_43.aln.fasta.contree
 â”‚   â”‚   â”œâ”€â”€ ğŸŒ³ 489mpoxsup90_348_98_43.aln.fasta.iqtree
 â”‚   â”‚   â”œâ”€â”€ ğŸŒ³ 489mpoxsup90_348_98_43.aln.fasta.treefile
 â”‚   â”‚   â”œâ”€â”€ ğŸ“ 489mpoxsup90_348_98_43.aln.fasta.log
 â”‚   â”‚   â””â”€â”€ ğŸŒ€ 489mpoxsup90_348_98_43.aln.fasta.{EXT}
 â”‚   â””â”€â”€ ğŸ“‚ auspice-augur_snakemake-workflow/
 â”‚Â       â”œâ”€â”€ ğŸ“‚ auspice/
 â”‚       â”‚   â””â”€ âš™ï¸  auspice_{DATASET}.json [Drag & Drop here: https://auspice.us/]
 â”‚Â       â”œâ”€â”€ ğŸ“‚ {DATASET}/ [Clade-I_Mpox-virus_genomic-diversity_DRC]
 â”‚       â”‚   â”œâ”€â”€ ğŸ“‚ tools-logfiles-summaries
 â”‚       â”‚   â”‚   â””â”€ ğŸ“ {XX}_{TOOL}.log
 â”‚       â”‚   â”œâ”€â”€ ğŸ“‚ genes-translations
 â”‚       â”‚   â”‚   â””â”€ ğŸ§¬ MPXV_gp{XXX}.fasta
 â”‚       â”‚   â”œâ”€â”€ ğŸŒ€ aa_muts.jsonbranch_lengths.json
 â”‚       â”‚   â”œâ”€â”€ ğŸŒ€ nt_muts.json
 â”‚       â”‚   â”œâ”€â”€ ğŸ§¬ sequences_aligned.fasta
 â”‚       â”‚   â”œâ”€â”€ ğŸ“ sequences_aligned.fasta.log
 â”‚       â”‚   â”œâ”€â”€ ğŸ§¬ sequences_filtered.fasta
 â”‚       â”‚   â”œâ”€â”€ ğŸ“Š sequences_index.tsv
 â”‚       â”‚   â”œâ”€â”€ ğŸ“ sequences_masked-delim.fasta.log
 â”‚       â”‚   â”œâ”€â”€ ğŸ“ sequences_masked-delim.iqtree.log
 â”‚       â”‚   â”œâ”€â”€ ğŸ§¬ sequences_masked.fasta
 â”‚       â”‚   â”œâ”€â”€ ğŸŒ€ traits.json
 â”‚       â”‚   â”œâ”€â”€ ğŸ“„ traitsdivision.mugration_model.txt
 â”‚       â”‚   â”œâ”€â”€ ğŸ“„ traitslocation.mugration_model.txt
 â”‚       â”‚   â”œâ”€â”€ ğŸŒ³ tree_raw.nwk
 â”‚       â”‚   â”œâ”€â”€ ğŸŒ³ tree_refine.nwk 
 â”œâ”€      â”œâ”€â”€ ğŸ“‚ Reports/
 â”‚       â”‚   â”œâ”€â”€ âš™ï¸  config.log
 â”‚      Â â”‚   â”œâ”€â”€ ğŸ“ settings.log
 â”‚       â”‚   â”œâ”€â”€ ğŸ“‚ files-summaries
 â”‚       â”‚   â”‚   â””â”€ âš™ï¸  augur_files-summary.txt
 â”‚      Â â”‚   â””â”€â”€ ğŸ“‚ graphs/
 â”‚    Â   â”‚   Â    â”œâ”€â”€ ğŸ“ˆ augur_dag.{PNG/PDF} 
 â”‚       â”‚Â       â”œâ”€â”€ ğŸ“ˆ augurfilegraph.{PNG/PDF}
 â”‚       â”‚Â       â””â”€â”€ ğŸ“ˆ augur_rulegraph.{PNG/PDF}
 â”‚       â””â”€â”€ ğŸ“¦ Reports_archive.tar.gz
 â””â”€â”€ ğŸ“‚ scripts/
     â””â”€â”€ ğŸ“‚ auspice-augur_snakemake-workflow/
         â”œâ”€â”€ ğŸ–¥ï¸ï¸  Start_Nextstrain.sh 
         â”œâ”€â”€ ğŸ“š README.md
         â”œâ”€â”€ ğŸªª LICENSE
         â”œâ”€â”€ ğŸ“‚ configuration/
         â”‚   â””â”€â”€ âš™ï¸  config.yaml
         â”œâ”€â”€ ğŸ“‚ resources/
         â”‚Â    â”œâ”€â”€ ğŸ“‚ config/
         â”‚    â”‚   â”œâ”€â”€ âš™ï¸  auspice_config.json
         â”‚    â”‚   â”œâ”€â”€ ğŸ·ï¸  colors.tsv
         â”‚    â”‚   â”œâ”€â”€ ğŸš« exclude_file.txt
         â”‚    â”‚   â”œâ”€â”€ ğŸŒ lat_longs.tsv
         â”‚    â”‚   â””â”€â”€ ğŸ¦  mpox_reference_NC003310.gb
     Â    â”‚    â”œâ”€â”€ ğŸ“‚ data/
         â”‚    â”‚   â”œâ”€â”€ ğŸ“Š  metadata_{DATASET}.tsv
         â”‚    â”‚   â””â”€â”€ ğŸ§¬ sequences_{DATASET}.fasta
     Â    â”‚    â”œâ”€â”€ ğŸ“‚ demo-test/
         â”‚    â”‚   â”œâ”€â”€ ğŸ“Š  metadata_DATA-TEST_10seq-clade-I.tsv
         â”‚    â”‚   â””â”€â”€ ğŸ§¬ sequences_DATA-TEST_10seq-clade-I.fasta
     Â    â”‚    â””â”€â”€ ğŸ“‚ visuals/
         â”‚        â”œâ”€â”€ ğŸ“ˆ augur_filegraph.png
         â”‚        â””â”€â”€ ğŸ“ˆ augur_rulegraph.png
	     â””â”€â”€ ğŸ“‚ worflow/
             â”œâ”€â”€ ğŸ“‚ environments/
         Â    â”‚   â”œâ”€â”€ ğŸœ augur_v.26.0.0.yaml
         Â    â”‚   â”œâ”€â”€ ğŸœ auspice_v.2.58.0.yaml
             â”‚   â”œâ”€â”€ ğŸœ gblocks_v.0.91b.yaml
             â”‚   â”œâ”€â”€ ğŸœ nexstrain-cli_v.8.5.3.yaml
         Â    â”‚   â””â”€â”€ ğŸœ workflow-base_v.2024.08.yaml
         Â    â””â”€â”€ ğŸ“‚ snakefiles/
                 â””â”€â”€ ğŸ“œ augur.smk
```

## OTHERS ##

Git repo picture: by NIAID - Monkeypox Virus, CC BY 2.0, https://commons.wikimedia.org/w/index.php?curid=120962820
