# Clade I Mpox virus genomic diversity in the Democratic Republic of the Congo, 2018 - 2024: Predominance of Zoonotic Transmission #


## ABOUT ##

**429** newly generated **MPXV Clade-I** genomes from **DRC** from **2018 to 2024**.


## STATUS ##

We are pleased to let you know that our manuscript is now accepted in principle at **Cell**.  


## DATA ##

**NCBI**: [NCBI](https://www.ncbi.nlm.nih.gov/genbank/)  
**Embargo:** The data will be publicly available on NCBI after [2025-03-12] or upon publication of the associated article.  

**Git**: [https://forge.ird.fr/transvihmi/nfernandez/Clade-I_Mpox-virus_genomic-diversity_DRC](https://forge.ird.fr/transvihmi/nfernandez/Clade-I_Mpox-virus_genomic-diversity_DRC)  

## LICENSE ##

**Intellectual property** belongs to INRB and all involved partners.

The **data** and **associated files** are made available under a Creative Commons Attribution-NoDerivatives 4.0 International license  
**[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/)**

### >>> Key Points ###

- **Attribution**: You must give appropriate credit, provide a link to the license, and indicate if changes were made.
- **NonCommercial**: You may not use the material for commercial purposes.
- **NoDerivatives**: If you remix, transform, or build upon the material, you may not distribute the modified material.  
                     The work can only be shared in its original form, without any alterations.

_For more details, see the full license [here](https://creativecommons.org/licenses/by/4.0)._


### >>> Preprint ###

The copyright holder for this preprint is the author/funder, who has granted **medRxiv** a license to display the preprint in perpetuity. It is made available under a **[CC-BY 4.0](https://creativecommons.org/licenses/by/4.0)** international license.  


### >>> Data ###

**IMPORTANT:** Data are released **only for public health usage**!  
**Any use** in commercial, academic publications, or other purposes **requires prior authorization from the authors**!



## CITATION ##

### Cell ###
**Clade I Monkeypox virus genomic diversity in the Democratic Republic of the Congo, 2018 - 2024: Predominance of Zoonotic Transmission**  
**DOI**:[]()  
**Print**:[]()  

### BioRxiv ###
**Clade I Mpox virus genomic diversity in the Democratic Republic of the Congo, 2018 - 2024: Predominance of Zoonotic Transmission**  
**DOI**: [https://doi.org/10.1101/2024.08.13.24311951](https://doi.org/10.1101/2024.08.13.24311951)  
**Preprint**: [https://www.medrxiv.org/content/10.1101/2024.08.13.24311951v1.full.pdf](https://www.medrxiv.org/content/10.1101/2024.08.13.24311951v1.full.pdf)  


## CORRESPONDING AUTHORS ##

- Eddy Kinganda-Lusamaki [eddylusamaki@gmail.com](eddylusamaki@gmail.com)
- Martine Peeters [martine.peeters@ird.fr](martine.peeters@ird.fr)


## AUTHORS ##

**Eddy Kingada Lusamaki** ; Adrienne Amuri Aziza ; Gradi Luakanda Ndelemo ; Emmanuel Hasivirwe Vakaniaki ; Nicolas Fernandez Nunez ; Catherine B. Pratt ;
Prince Akil Bandali ; Jean-Claude Makangara Cigolo ; Nicole A. Hoff ; Sabin Sabiti Nundu ; Noella Mulopo Mukanya ; Michel Ngimba ; Brigitte Modrada Madakpa ;
Ruth Diavita ; Princesse Paku Tshambu ; Elisabeth Pukuta Simbu ; Sydney Merritt ; Aine O'Toole ; Nicola Low ; Antoine Nkuba Ndaye ; Hugo Kavunga Membo ;
Robert Shongo ; Emile Malembi ; Thierry Kalonji ; Cris Kacita ; Emmanuel Lokilo Lofiko ; Sifa Kavira Bapitani ; Ola Rilia ; Pauline-Chloe Muswamba ;
Hugo Kavunga ; Membo Francisca ; Muyembe Mawete ; Raphael Lumembe ; Gabriel Kabamba ; Michel Kenye ; Andre Citenga ; Fiston Cikaya ; Zadok Yungwe ;
Meris Matondo ; Francois Kasongo ; Paul Tshiminyi ; Junior Bulabula ; Frida Nkawa ; Emile Muhindo Milonde ; Adele Kavira Kamaliro ; Zephanie Paluku Kalimuli ;
Jeriel Mubalaa Mufungizi ; Tavia Bodisa Matamu ; Mike R. Wiley ; Elise Devos ; Eugene Bangwen ; Isabel Brosius ; Edyth Parker ; Genay Pilarowski ;
Laurens Liesenborghs ; Tony Wawina Bokalanga ; Koen Vercauteren ; Daniel Mukadi Bamuleka ; Lorenzo Subissi ; Jean-Jacques Muyembe ; Jason Kindrachuk ;
Emilande Guichet ; Ahidjo Ayouba ; Andrew Rambaut ; Eric Delaporte ; Sofonias K. Tessema ; Eric D'Ortenzio ; Anne W. Rimoin ; Lisa E. Hensley ;
Placide Mbala Kingebeni ; Martine Peeters ; Steve Ahuka Mundeke

## Rulegraph ##

<img src="./scripts/auspice-augur_snakemake-workflow/resources/visuals/augur_rulegraph.png" width="100" height="300">  

<img src="./scripts/auspice-augur_snakemake-workflow/resources/visuals/augur_filegraph.png" width="200" height="600">  

## Auspice ##

**Drag & Drop** [auspice.json](./results/auspice-augur_snakemake-workflow/auspice/auspice_Clade-I_Mpox-virus_genomic-diversity_DRC.json) here: [asupice.us](https://auspice.us)


## Git map ##

```shell
 🧩 GIT/
 ├── 📚 README.md [READ FIRST]
 ├── 🪪 LICENSE
 ├── 🚫 .gitignore
 ├── 📂 .git/
 ├── 📂 data/
 │   ├── 📊  sequences_metadata.tsv [COMING SOON]
 │   ├── 📂 sequences_fasta_pre-release/ [429 newly sequenced]
 │   │   └── 🧬 {SAMPLE}.fasta
 │   └── 📂 auspice-augur_snakemake-workflow/ [fasta and metadata for auspice figure]
 │       ├── 📊  metadata_Clade-I_Mpox-virus_genomic-diversity_DRC.tsv
 │       └── 🧬 sequences_Clade-I_Mpox-virus_genomic-diversity_DRC.fasta
 ├── 📂 publications/
 │   └── 📝 Pre-print\_bioRxiv\_Clade-I\_Mpox-virus\_genomic-diversity\_DRC\_2018-2024\_Predominance-Zoonotic-Transmission.pdf
 ├── 📂 results/
 │   ├── 📂 apobec3_analysis/
 │   │   ├── 🧬 530mpox429_98_3.fasta
 │   │   ├── 🌳 530mpox429_98_3.tree
 │   │   ├── 🧬 530mpox429_98_3.aln.fasta
 │   │   ├── 🌳 530mpox429_98_3.aln.fasta.iqtree
 │   │   ├── 🌳 530mpox429_98_3.aln.fasta.treefile
 │   │   ├── 📊 530mpox429_98_3.tree.amino_acid.reconstruction.csv
 │   │   ├── 📊 530mpox429_98_3.tree.branch_snps.reconstruction.csv
 │   │   ├── 📊 530mpox429_98_3.tree.branch_snps.reconstruction.csv
 │   │   ├── 📝 530mpox429_98_3.aln.fasta.log
 │   │   └── 🌀 530mpox429_98_3.aln.fasta.{EXT}
 │   ├── 📂 ML_tree/
 │   │   ├── 🧬 489mpoxsup90_348_98_43.aln.fasta
 │   │   ├── 🌳 489mpoxsup90_348_98_43.aln.fasta.contree
 │   │   ├── 🌳 489mpoxsup90_348_98_43.aln.fasta.iqtree
 │   │   ├── 🌳 489mpoxsup90_348_98_43.aln.fasta.treefile
 │   │   ├── 📝 489mpoxsup90_348_98_43.aln.fasta.log
 │   │   └── 🌀 489mpoxsup90_348_98_43.aln.fasta.{EXT}
 │   └── 📂 auspice-augur_snakemake-workflow/
 │       ├── 📂 auspice/
 │       │   └─ ⚙️  auspice_{DATASET}.json [Drag & Drop here: https://auspice.us/]
 │       ├── 📂 {DATASET}/ [Clade-I_Mpox-virus_genomic-diversity_DRC]
 │       │   ├── 📂 tools-logfiles-summaries
 │       │   │   └─ 📝 {XX}_{TOOL}.log
 │       │   ├── 📂 genes-translations
 │       │   │   └─ 🧬 MPXV_gp{XXX}.fasta
 │       │   ├── 🌀 aa_muts.jsonbranch_lengths.json
 │       │   ├── 🌀 nt_muts.json
 │       │   ├── 🧬 sequences_aligned.fasta
 │       │   ├── 📝 sequences_aligned.fasta.log
 │       │   ├── 🧬 sequences_filtered.fasta
 │       │   ├── 📊 sequences_index.tsv
 │       │   ├── 📝 sequences_masked-delim.fasta.log
 │       │   ├── 📝 sequences_masked-delim.iqtree.log
 │       │   ├── 🧬 sequences_masked.fasta
 │       │   ├── 🌀 traits.json
 │       │   ├── 📄 traitsdivision.mugration_model.txt
 │       │   ├── 📄 traitslocation.mugration_model.txt
 │       │   ├── 🌳 tree_raw.nwk
 │       │   ├── 🌳 tree_refine.nwk 
 ├─      ├── 📂 Reports/
 │       │   ├── ⚙️  config.log
 │       │   ├── 📝 settings.log
 │       │   ├── 📂 files-summaries
 │       │   │   └─ ⚙️  augur_files-summary.txt
 │       │   └── 📂 graphs/
 │       │       ├── 📈 augur_dag.{PNG/PDF} 
 │       │       ├── 📈 augurfilegraph.{PNG/PDF}
 │       │       └── 📈 augur_rulegraph.{PNG/PDF}
 │       └── 📦 Reports_archive.tar.gz
 └── 📂 scripts/
     └── 📂 auspice-augur_snakemake-workflow/
         ├── 🖥️️  Start_Nextstrain.sh 
         ├── 📚 README.md
         ├── 🪪 LICENSE
         ├── 📂 configuration/
         │   └── ⚙️  config.yaml
         ├── 📂 resources/
         │    ├── 📂 config/
         │    │   ├── ⚙️  auspice_config.json
         │    │   ├── 🏷️  colors.tsv
         │    │   ├── 🚫 exclude_file.txt
         │    │   ├── 🌍 lat_longs.tsv
         │    │   └── 🦠 mpox_reference_NC003310.gb
         │    ├── 📂 data/
         │    │   ├── 📊  metadata_{DATASET}.tsv
         │    │   └── 🧬 sequences_{DATASET}.fasta
         │    ├── 📂 demo-test/
         │    │   ├── 📊  metadata_DATA-TEST_10seq-clade-I.tsv
         │    │   └── 🧬 sequences_DATA-TEST_10seq-clade-I.fasta
         │    └── 📂 visuals/
         │        ├── 📈 augur_filegraph.png
         │        └── 📈 augur_rulegraph.png
	     └── 📂 worflow/
             ├── 📂 environments/
             │   ├── 🍜 augur_v.26.0.0.yaml
             │   ├── 🍜 auspice_v.2.58.0.yaml
             │   ├── 🍜 gblocks_v.0.91b.yaml
             │   ├── 🍜 nexstrain-cli_v.8.5.3.yaml
             │   └── 🍜 workflow-base_v.2024.08.yaml
             └── 📂 snakefiles/
                 └── 📜 augur.smk
```

## OTHERS ##

Git repo picture: by NIAID - Monkeypox Virus, CC BY 2.0, https://commons.wikimedia.org/w/index.php?curid=120962820
